src/Compilers/CSharp/Test/Symbol/Compilation/UsedAssembliesTests.cs

```diff
@@ -105,13 +118,18 @@ private void AssertUsedAssemblyReferences(Compilation comp, MetadataReference[]
 
         private void AssertUsedAssemblyReferences(Compilation comp, params MetadataReference[] expected)
         {
-            AssertUsedAssemblyReferences(comp, expected, new DiagnosticDescription[] { }, new DiagnosticDescription[] { });
+            AssertUsedAssemblyReferences(comp, expected, new DiagnosticDescription[] { }, new DiagnosticDescription[] { }, specificReferencesToAssert: null);
+        }
+
+        private void AssertUsedAssemblyReferences(Compilation comp, MetadataReference[] expected, MetadataReference[] specificReferencesToAssert)
+        {
+            AssertUsedAssemblyReferences(comp, expected, new DiagnosticDescription[] { }, new DiagnosticDescription[] { }, specificReferencesToAssert);
         }
 
         private Compilation AssertUsedAssemblyReferences(string source, MetadataReference[] references, params MetadataReference[] expected)
```

Comment:
>AssertUsedAssemblyReferences [](start = 28, length = 28)

nit: it's odd to have an Assert method return a result. Consider renaming. Also, I was generally a bit confused about the many overloads, especially with use of `params`. It's probably okay within IDE, but harder as text review.